chainer/cuda.py

```diff
@@ -555,6 +559,8 @@ def get_array_module(*args):
 
     """
     if available:
+        args = (arg.data if isinstance(arg, chainer.Variable) else arg
```

Comment:
Please use list for performance.
`args = [arg.data if instance(arg, chainer.Variable) else arg for arg in args]`.